openupgradelib/openupgrade.py

```diff
@@ -643,13 +644,16 @@ def write_value(ids, field, value):
                 cr.execute(query, (params[0], sub_ids))
 
     for model in default_spec.keys():
-        obj = pool.get(model)
-        if not obj:
+        obj = pool.get(model, False)
```

Comment:
I just checked with

```shell
grep -R use_env addons/*/migrations --files-with-matches | xargs grep --after-context=2 set_defaults
addons/point_of_sale/migrations/9.0.1.0.1/post-migration.py:    openupgrade.set_defaults(env.cr, env.registry, column_defaults)
addons/point_of_sale/migrations/9.0.1.0.1/post-migration.py-    update_barcodes_nomenclatures(env)
addons/point_of_sale/migrations/9.0.1.0.1/post-migration.py-    openupgrade.load_data(
--
addons/website/migrations/9.0.1.0/post-migration.py:    openupgrade.set_defaults(
addons/website/migrations/9.0.1.0/post-migration.py-        env.cr, pool, {'website': [('cdn_filters', None)]}, use_orm=True,
addons/website/migrations/9.0.1.0/post-migration.py-    )
```

that this will not break in the <= 9.0, but that's only coincidence. But on the other hand, we can also just see it as mistake to pass an env in <= 10

Reply:
Why is a mistake to pass an env? I don't understand.

Reply:
Fine by me to suppose that pool is not an env before 10.0, especially as it is documented above.